sticht/slack.py

```diff
@@ -87,7 +91,7 @@ async def get_slack_events():
         logging.error('Scribereader unavailable. Not tailing slack events.')
         return
 
-    host_and_port = scribereader.get_env_scribe_host(SCRIBE_ENV, True)
+    host_and_port = scribereader.get_tail_host_and_port(SCRIBE_ENV, True)
```

Comment:
`get_tail_host_and_port` returns a `host, port` tuple not a dict.

Reply:
thx